subprojects/test-kit/src/main/java/org/gradle/testkit/runner/internal/DefaultGradleRunner.java

```diff
@@ -62,6 +62,9 @@
     private InputStream standardInput;
     private boolean forwardingSystemStreams;
 
+    //TODO is it OK to configure Tooling API/TestKit with empty Map by default? Is it a compatible change?
```

Comment:
If we don't have a test for that yet, we should create one. By default the environment variables should be the same as the calling process. This should not be changed.

Reply:
Thank you for review!